CMakeLists.txt

```diff
@@ -83,6 +89,41 @@ else()
     message(FATAL_ERROR "Unknown OS type")
 endif()
 
+if(NOT UMF_LINK_HWLOC_STATICALLY)
+    pkg_check_modules(LIBHWLOC hwloc>=2.3.0)
+    if(NOT LIBHWLOC_FOUND)
+        find_package(LIBHWLOC 2.3.0 REQUIRED hwloc)
+    endif()
+    # add PATH to DLL on Windows
+    set(DLL_PATH_LIST
+        "${DLL_PATH_LIST};PATH=path_list_append:${LIBHWLOC_LIBRARY_DIRS}/../bin"
+    )
+else()
+    include(FetchContent)
+    FetchContent_Declare(
+        hwloc_targ
+        GIT_REPOSITORY "https://github.com/open-mpi/hwloc.git"
+        GIT_TAG master
+        FIND_PACKAGE_ARGS)
+
+    FetchContent_GetProperties(hwloc_targ)
+    if(NOT hwloc_targ_POPULATED)
+        FetchContent_MakeAvailable(hwloc_targ)
+    endif()
+
+    set(hwloc_cmake ${hwloc_targ_SOURCE_DIR}/contrib/windows-cmake)
```

Comment:
Is this directory available both on Linux and Windows?

Reply:
this path is only to be used on Windows - I don't think we even need to test this for Linux. @KFilipek could you add an additional "AND WINDOWS" to L#92 with an appropriate comment?

Reply:
We discussed it offline, only valid way to use it is Windows with release build, because of the requirement to not add extra CMake step with dependency preparation.
Done.